conftest.py

```diff
@@ -176,37 +168,61 @@ def function_to_seed_torch(seed):
 
 
 @pytest.fixture(autouse=True)
-def autoseeding_of_everything(record_property, request):
+def autoseeding_of_everything(request):
     """Seed everything we can, for determinism."""
-    main_seed = request.config.getoption("--forcing_random_seed", default=None)
+    randomly_seed = request.config.getoption("--randomly-seed", default=None)
 
-    if main_seed is None:
-        main_seed = random.randint(0, 2**64 - 1)
+    if randomly_seed is None:
+        raise ValueError("why isn't --randomly-seed set?")
```

Comment:
would prefer a message like `"Please set --randomly-seed set with an integer"`

Reply:
But they don't have to set this themselves, it's done by the system. In other words, it's normally impossible to raise this error

Reply:
I can change the sentence if you want, but not to what you mentioned because it's misleading

Reply:
hmm, so you're saying running `pytest ... --randomly-seed` with nothing after won't trigger this issue ? 

if so ok but yes maybe put it more like `--randomly-seed has not been set properly` then

Reply:
no it fails, but it's already caught by the system, much before, directly in randomly-seed or pytest:

```
python -m pytest -xsvv "tests/common/test_utils.py::test_compute_bits_precision[x0-8]" --randomly-seed     
/Users/benoitchevalliermames/Documents/Zama/Git/concrete-ml/.venv/lib/python3.9/site-packages/_pytest/config/__init__.py:330: PluggyTeardownRaisedWarning: A plugin raised an exception during an old-style hookwrapper teardown.
Plugin: helpconfig, Hook: pytest_cmdline_parse
UsageError: usage: __main__.py [options] [file_or_dir] [file_or_dir] [...]
__main__.py: error: argument --randomly-seed: expected one argument
For more information see https://pluggy.readthedocs.io/en/stable/api_reference.html#pluggy.PluggyTeardownRaisedWarning
  config = pluginmanager.hook.pytest_cmdline_parse(
ERROR: usage: __main__.py [options] [file_or_dir] [file_or_dir] [...]
__main__.py: error: argument --randomly-seed: expected one argument
```

Reply:
got it, so yeah maybe just update the message and that should be enough !

Reply:
I was more referring to my second suggestion : `--randomly-seed has not been set properly` as clearly it's not a user problem !

Reply:
but the user doesn't have to set --randomly-seed
to me, it's more an internal issue, like "I arrive in an impossible situation", we should have an issue like that

Reply:
yes I know, if you prefer maybe `--randomly-seed has not been properly configured internally.`

Reply:
if you want: done